GithubAppPostInstall.md

```diff
@@ -0,0 +1,5 @@
+# Github App Post Installation page
```

Comment:
When someone installs the Github App, they will be redirected to this stub page.